### PR TITLE
Fix intermittent test failure

### PR DESCRIPTION
**Pull request by julenka**
Two fixes:

1. Fixing failure `Microsoft.MixedReality.Toolkit.Tests.BaseCursorTests.CursorContextMove`, happened about 2/100 times. Issue seems similar -- Unity sometimes doesn't raycast when there's just one collider in scene. With a second collider, tests pass 100/100

2. Fix failure `Microsoft.MixedReality.Toolkit.Tests.InteractableTests.TestInputActionMenuInput` by adding function that ensures input system updates (all input events raised can get received). Also make the test easier to diagnose by also checking if input has gone down, and up.

## Changes
- Fixes: another failure in #6058

### CODE REVIEW

File: Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs [hunk 1]
```diff
@@ -358,6 +358,22 @@ internal static IEnumerator WaitForEnterKey()
             }
         }
 
+        /// <summary>
+        /// Sometimes it take a few frames for inputs raised via InputSystem.OnInput*
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs]: Huh, that seems concerning. Do you have any ideas why there might be a few frames of lag?

Comment (julenka) [reply Assets/MixedRealityToolkit.Tests/PlayModeTests/PlayModeTestUtilities.cs]: No, I wasn't sure how to debug that. Any ideas?

**Pull request merged**